charts/virtual-kubelet/templates/deployment.yaml

```diff
@@ -60,13 +60,17 @@ spec:
           value: {{ .aciResourceGroup }}
         - name: ACI_REGION
           value: {{ .aciRegion }}
+        - name: ACI_EXTRA_USER_AGENT
+          value: {{ printf "helm-chart/aks/%s/%s" $.Chart.Name $.Chart.Version }}
```

Comment:
This doesn't seem right?

Reply:
anything wrong? I've verified the package locally.

Reply:
Why are we adding aks to the generic template?

Reply:
L45, it has the "targetAKS" flag == true

Reply:
we will deprecate the virtual-kubelet-for-aks helm chart soon.